ONI-DenseLogic/RemapperSideScreen.cs

```diff
@@ -135,5 +164,75 @@ private sealed class BitOption : IListableOption {
 				return name;
 			}
 		}
+
+		private sealed class RemapperRow {
+			private RemapperSideScreen parent;
+
+			/// <summary>
+			/// The panel containing this row
+			/// </summary>
+			public PPanel Row { get; }
+
+			public RemapperRow(int index, RemapperSideScreen parent) {
+				// TODO This could probably be done as a relative layout eventually, but
+				// until PRelativePanel is added this is the best we can do
+				this.parent = parent;
+				Row = new PPanel("RemapperRow") {
+					Margin = new RectOffset(4, 4, 4, 4),
+					FlexSize = new Vector2(1.0f, 0.0f),
+				};
+				var RowBackground = new PPanel("RemapperRowBackground") {
+					FlexSize = new Vector2(1.0f, 0.0f)
+				};
+				RowBackground.OnRealize += gameObject => {
+					var img = gameObject.AddComponent<KImage>();
+					img.sprite = PUITuning.Images.GetSpriteByName("BitSelectorSideScreenRow");
+					img.type = Image.Type.Sliced;
+					img.color = new Color(1.0f, 1.0f, 1.0f);
+				};
+				Row.AddChild(RowBackground);
+				var RowInternal = new PPanel("RemapperRowInternal") {
+					Margin = new RectOffset(4, 4, 4, 4),
+					FlexSize = new Vector2(1.0f, 0.0f),
+				};
+				RowInternal.OnRealize += gameObject => {
+					var img = gameObject.AddComponent<KImage>();
+					img.sprite = PUITuning.Images.GetSpriteByName("overview_highlight_outline_sharp");
+					img.type = Image.Type.Sliced;
+					img.color = new Color(0.898f, 0.898f, 0.898f);
+				};
+				RowBackground.AddChild(RowInternal);
+				var RowInternalGrid = new PGridPanel("RemapperRowInternalGrid") {
+					Margin = new RectOffset(4, 4, 4, 4),
+					FlexSize = new Vector2(1.0f, 0.0f),
+				};
+				RowInternalGrid.AddRow(new GridRowSpec());
```

Comment:
Triply nested panels, unless frozen, are really hard on the Unity layout engine. Is there any way we could restructure the UI to have a simpler hierarchy?